api/proxy/servers/rest/server.go

```diff
@@ -18,11 +18,23 @@ import (
 	"github.com/gorilla/mux"
 )
 
+// PubliclyExposedInfo ... PubliclyExposedInfo is used by the /info endpoint
+type PubliclyExposedInfo struct {
```

Comment:
any value in adding `ReadOnlyMode` as a field here for a batch poster compatibility check?